linux_os/guide/system/accounts/accounts-banners/gui_login_banner/dconf_gnome_banner_enabled/ansible/shared.yml

```diff
@@ -5,7 +5,7 @@
 # disruption = medium
 - name: "Enable GNOME3 Login Warning Banner"
   ini_file:
-    dest: "/etc/dconf/db/local.d/00-security-settings"
+    dest: "/etc/dconf/db/gdm.d/00-security-settings"
```

Comment:
This needs to be changed also for the lock from `/etc/dconf/db/local.d/locks/00-security-settings-lock` to `/etc/dconf/db/gdm.d/locks/00-security-settings-lock`.

Reply:
Nice catch, fixed by rebase.